src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -293,6 +293,34 @@
         text: az aks update -g MyResourceGroup -n MyManagedCluster --load-balancer-outbound-ip-prefixes <ip-prefix-resource-id-1,ip-prefix-resource-id-2>
 """
 
+helps['aks kollect'] = """
+    type: command
+    short-summary: Collecting diagnostic information for the Kubernetes cluster.
+    long-summary: |-
+        Collecting diagnostic information for the Kubernetes cluster and store them in the specified storage account.
+        You can provider the storage account in three ways:
+          storage account name and a shared access signature with write permission.
+          resource Id to a storage account you own.
+          the storagea account in diagnostics settings for your managed cluster.
+    parameters:
+        - name: --storage-account-name
+          type: string
+          short-summary: The name of the storage account to save the diagnostic information.
+        - name: --sas-token
+          type: string
+          short-summary: The SAS token with writable permission for the storage account.
```

Comment:
pls check the best practice here:

https://github.com/Azure/azure-cli-extensions/pull/947/files

storage_account parameter is an example